config/cloud.cfg.tmpl

```diff
@@ -70,8 +70,8 @@ cloud_init_modules:
 {% endif %}
  - set_hostname
  - update_hostname
-{% if variant not in ["freebsd", "netbsd"] %}
  - update_etc_hosts
+{% if variant not in ["freebsd", "netbsd"] %}
```

Comment:
i'm pretty, and sure, the stuff below doesn't work on OpenBSD (out of the box) either, so we can add that to the list. or say something like

```suggestion
{% if variant.endswith("bsd") %}
```